sessions/R-estimation-and-the-renewal-equation.qmd

```diff
@@ -144,8 +144,8 @@ gen_time_pmf <- gen_time_pmf / sum(gen_time_pmf) ## renormalise
 As always we first load the stan model and spend some time trying to understand it.
 
 ```{r stan_estimate_r}
-mod <- cmdstan_model(here("stan", "estimate-r.stan"))
-mod
+r_mod <- cmdstan_model(here("stan", "estimate-r.stan"))
+r_mod
```

Comment:
the renaming here doesn't happening when the model is fit (line 175?). I think it needs to be the same for both?

Reply:
yep looks like this was introduced by merging in `main`, fixing

Reply:
sorry!